pkg/webhook/preflight/generic/spec.go

```diff
@@ -45,7 +45,8 @@ func newConfigurationCheck(
 		configurationCheck.result.InternalError = true
 		configurationCheck.result.Causes = append(configurationCheck.result.Causes,
 			preflight.Cause{
-				Message: fmt.Sprintf("Failed to unmarshal cluster variable %s: %s",
+				Message: fmt.Sprintf(
+					"Failed to unmarshal cluster variable %q: %s. This error should not happen under normal circumstances. Please report it.", ///nolint:lll // Message is long.
```

Comment:
an unmarshal failure implies that the underlying manifest has an issue (usually indentations, extraneous characters, etc.) The prompt to report it seems excessive. We should be prompting them to revisit the manifest.

Reply:
Addressed in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/1210/commits/4023398b85ab2738872d2c96bf66546dcfaa82f0 and https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/1210/commits/4e94afe67e72402596f6528b76df211d8a47d069